src/amo/components/AddonAdminLinks/index.js

```diff
@@ -106,14 +109,17 @@ export class AddonAdminLinksBase extends React.Component<InternalProps> {
           className="AddonAdminLinks-contentReview-link"
           href={`/reviewers/review-content/${addon.slug}`}
         >
-          {i18n.gettext('Content review add-on')}
+          {// translators: This action allows a user to perform a content review of an add-on.
```

Comment:
Does this work? I'm not sure if it does.

```js
{/* translators: This action allows a user to perform a content review of an add-on. */}
{i18n.gettext('Content review add-on')}
```

Reply:
I tested your syntax and it doesn't seem to work for extraction.